apis/oam/v1alpha2/core_workload_types.go

```diff
@@ -0,0 +1,398 @@
+/*
```

Comment:
We'd better not put all core workload in one file. A dir is better instead.

Could you rename this file from core_workload_types.go to containerized_workload_types.go ?

Reply:
Can we worry about this when more than one workload kind exists?

Reply:
Sure ;)